src/commands/openapi/upload.ts

```diff
@@ -137,7 +142,7 @@ export default class OpenAPIUploadCommand extends BaseCommand<typeof OpenAPIUplo
     if (method === 'PUT') {
       // bypass the prompt if we're in a CI environment
       prompts.override({
-        confirm: isCI() ? true : undefined,
+        confirm: isCI() ? true : this.flags.overwrite,
```

Comment:
I _think_ this is might be a bug but I might be wrong. I believe if `this.flags.overwrite` is `false`, the value of `confirm` is overridden to `false`, which skips the prompt and throws the error on line 155. Can you confirm? I believe this is how we'd fix it:

```suggestion
        confirm: isCI() || this.flags.overwrite ? true : undefined,
```

Reply:
@kanadgupta 

I made the recommended change (I had to modify the arg name), and it worked as desired — 

1. when the flag is provided, the prompt is skipped
2. when the flag is omitted, the prompt is shown